dashboard/config/locales/en.yml

```diff
@@ -321,6 +321,7 @@ en:
       sections: "Sections"
       help_support: "Help and support"
       report_bug: "Report a problem"
+      report_abuse: "Report abuse"
```

Comment:
Do these strings also need to be added to pegasus? The whole `loc_prefix` thing in the hamburger menu code reflects the fact that the component is used in both places

Reply:
Oh good catch. I thought I had already added them but when I checked the i18n google sheet, they weren't there. I will need to run a sync in & up, right?

Reply:
hah, that would make more sense. Unfortunately, the pegasus gsheet is managed by the staging server, and changes get consumed with the staging content scoop